components/widgets/menu/menu_items/menu_item_toggle_modal_redux.tsx

```diff
@@ -17,17 +17,17 @@ type Props = {
     dialogProps?: Dictionary<any>;
     extraText?: string;
     text?: React.ReactNode;
-    ariaLabel?: string;
+    accessibilityLabel?: string;
     className?: string;
     children?: React.ReactNode;
     sibling?: React.ReactNode;
     showUnread?: boolean;
 }
 
-export const MenuItemToggleModalReduxImpl: React.FC<Props> = ({modalId, dialogType, dialogProps, text, ariaLabel, extraText, children, className, sibling, showUnread}: Props) => (
+export const MenuItemToggleModalReduxImpl: React.FC<Props> = ({modalId, dialogType, dialogProps, text, accessibilityLabel, extraText, children, className, sibling, showUnread}: Props) => (
     <>
         <ToggleModalButtonRedux
-            accessibilityLabel={ariaLabel}
+            accessibilityLabel={accessibilityLabel}
```

Comment:
Other tests still failed in Cypress.
<img width="1153" alt="Screen Shot 2021-09-30 at 4 09 30 AM" src="https://user-images.githubusercontent.com/5334504/135341314-c91cbddc-3897-4ddd-87c7-5ddb55c6fe91.png">
<img width="1118" alt="Screen Shot 2021-09-30 at 4 09 55 AM" src="https://user-images.githubusercontent.com/5334504/135341321-b6524879-4ce4-4cbc-aaaa-ce6ddc4c6fb8.png">

Reply:
Confirmed. I'm still seeing the same issue.
<img width="1036" alt="Screen Shot 2021-10-06 at 9 40 14 PM" src="https://user-images.githubusercontent.com/5334504/136214061-e5b47ddd-56f5-438d-80d5-0594284d62e5.png">